### DISCUSSION TITLE
Scrobbler "There was a problem contacting Last.fm"

### DISCUSSION THREAD

**Discussion by CwalkPinoy** [audacious-media-player/audacious#1373] [General]
I just recompiled audacious-plugins on gentoo with the scrobbler option. When going to plugins and trying to click "check permission" I get an error box. Revoke Permissions does nothing.
Flatpak works fine.
Audacious -V reports the following when I click that box:

Entity: line 5: parser error : Extra content at the end of the document

^

**Comment by radioactiveman** [comment]
This issue will be fixed in Audacious 4.4. You can use the following commits for 4.3.1:
- https://github.com/audacious-media-player/audacious-plugins/commit/c9246b1c8b09553250c40d222115c2c0865779e2
- https://github.com/audacious-media-player/audacious-plugins/commit/5103ba0f504f1a98cac7babca3a0e45ca969f7f8
